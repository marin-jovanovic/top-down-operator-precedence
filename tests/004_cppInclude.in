cpp_include "t_1"
