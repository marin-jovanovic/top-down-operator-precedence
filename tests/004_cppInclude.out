Document
 HeaderManager
  Header
   CppInclude
    "cpp_include"
    Literal
     "t_1"
  HeaderManager
   end of header instructions
 DefinitionManager
  end of definition instructions
